opensrp-chw/build.gradle

```diff
@@ -346,13 +346,13 @@ android {
             buildConfigField "String", 'DEFAULT_LOCATION_DEBUG', '"CHA"'
             buildConfigField "int", "MAX_CONNECTION_TIMEOUT", '5'
             buildConfigField "int", "MAX_READ_TIMEOUT", '5'
-            buildConfigField "int", "DATABASE_VERSION", '8'
+            buildConfigField "int", "DATABASE_VERSION", '9'
```

Comment:
Kindly update this also https://github.com/opensrp/opensrp-client-chw/blob/master/opensrp-chw/src/liberia/java/org/smartregister/chw/repository/ChwRepositoryFlv.java#L49